Header Header
uint8 id
string type
time start_time
time end_time
string data